eslint.config.mjs

```diff
@@ -1,24 +1,58 @@
+import { defineConfig } from 'eslint/config'
 import globals from 'globals'
 import eslintPluginEslintPlugin from 'eslint-plugin-eslint-plugin/configs/all'
 import eslintPluginJsonc from 'eslint-plugin-jsonc'
 import eslintPluginNodeDependencies from 'eslint-plugin-node-dependencies'
 import eslintPluginPrettierRecommended from 'eslint-plugin-prettier/recommended'
 import eslintPluginUnicorn from 'eslint-plugin-unicorn'
+import eslintMarkdown from '@eslint/markdown'
+import markdownPreferences from 'eslint-plugin-markdown-preferences'
```

Comment:
Please rename this variable to `eslintPluginMarkdownPreferences` for consistency, even if it's long.